app/models/manageiq/providers/vmware/cloud_manager/orchestration_template.rb

```diff
@@ -112,6 +112,15 @@ def vm_param_groups(template)
           :constraints   => [
             OrchestrationTemplate::OrchestrationParameterBoolean.new
           ]
+        ), OrchestrationTemplate::OrchestrationParameter.new(
+          :name          => param_name('guest_customization', [vm_idx]),
+          :label         => 'Guest customization',
+          :description   => 'Allow guest customization',
```

Comment:
Description should be something more meaningful, please see how it's described on the vCD dashboard.

![capture](https://user-images.githubusercontent.com/8102426/37826508-fceb2f8c-2e93-11e8-9bcb-f7ecca53ee22.PNG)